cupyx/scipy/signal/_upfirdn.py

```diff
@@ -492,7 +492,7 @@ def upfirdn(
     .. [1] P. P. Vaidyanathan, Multirate Systems and Filter Banks,
        Prentice Hall, 1993.
     """
```

Comment:
Could you add these lines to ensure backward compatibility?
```suggestion
    """
    if mode is None:
        mode = "constant"
```

Reply:
LGTM excepts this comment.

Reply:
Done